From f780419a57072bf654cecf030829f4888d9208a6 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Tue, 10 Sep 2019 20:45:00 +0900
Subject: [PATCH 1/9] shared: rename file ringlatencystats to be short

Filename of `ringlatencystats.c` and its header is not understandable
because it is too long and has no separators. `latency_stats` is enough
for the usage and no need to add `ring` explicitly.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/mirror/Makefile                                             | 2 +-
 src/mirror/spp_mirror.c                                         | 2 +-
 src/pcap/Makefile                                               | 2 +-
 src/pcap/spp_pcap.c                                             | 2 +-
 .../spp_worker_th/{ringlatencystats.c => latency_stats.c}       | 2 +-
 .../spp_worker_th/{ringlatencystats.h => latency_stats.h}       | 0
 src/shared/secondary/spp_worker_th/port_capability.c            | 2 +-
 src/vf/Makefile                                                 | 2 +-
 src/vf/classifier.c                                             | 2 +-
 src/vf/forwarder.c                                              | 2 +-
 src/vf/spp_vf.c                                                 | 2 +-
 11 files changed, 10 insertions(+), 10 deletions(-)
 rename src/shared/secondary/spp_worker_th/{ringlatencystats.c => latency_stats.c} (99%)
 rename src/shared/secondary/spp_worker_th/{ringlatencystats.h => latency_stats.h} (100%)

diff --git a/src/mirror/Makefile b/src/mirror/Makefile
index ed2cb33..0adab15 100644
--- a/src/mirror/Makefile
+++ b/src/mirror/Makefile
@@ -26,7 +26,7 @@ SRCS-y += $(SPP_WKT_DIR)/port_capability.c
 SRCS-y += $(SPP_WKT_DIR)/conn_spp_ctl.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_parser.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_runner.c
-SRCS-y += $(SPP_WKT_DIR)/ringlatencystats.c
+SRCS-y += $(SPP_WKT_DIR)/latency_stats.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_res_formatter.c
 
 CFLAGS += -DALLOW_EXPERIMENTAL_API
diff --git a/src/mirror/spp_mirror.c b/src/mirror/spp_mirror.c
index 0ace09c..40fbc6f 100644
--- a/src/mirror/spp_mirror.c
+++ b/src/mirror/spp_mirror.c
@@ -21,7 +21,7 @@
 #include "shared/secondary/spp_worker_th/port_capability.h"
 
 #ifdef SPP_RINGLATENCYSTATS_ENABLE
-#include "shared/secondary/spp_worker_th/ringlatencystats.h"
+#include "shared/secondary/spp_worker_th/latency_stats.h"
 #endif
 
 /* Declare global variables */
diff --git a/src/pcap/Makefile b/src/pcap/Makefile
index 5e6f0a6..b2ed5f0 100644
--- a/src/pcap/Makefile
+++ b/src/pcap/Makefile
@@ -24,7 +24,7 @@ SRCS-y += $(SPP_SEC_DIR)/utils.c
 SRCS-y += $(SPP_SEC_DIR)/string_buffer.c
 SRCS-y += $(SPP_WKT_DIR)/conn_spp_ctl.c
 SRCS-y += $(SPP_WKT_DIR)/port_capability.c
-SRCS-y += $(SPP_WKT_DIR)/ringlatencystats.c
+SRCS-y += $(SPP_WKT_DIR)/latency_stats.c
 
 CFLAGS += -DALLOW_EXPERIMENTAL_API
 CFLAGS += $(WERROR_FLAGS) -O3 -MMD
diff --git a/src/pcap/spp_pcap.c b/src/pcap/spp_pcap.c
index ac87bd3..ab4d796 100644
--- a/src/pcap/spp_pcap.c
+++ b/src/pcap/spp_pcap.c
@@ -24,7 +24,7 @@
 #include "shared/secondary/spp_worker_th/port_capability.h"
 
 #ifdef SPP_RINGLATENCYSTATS_ENABLE
-#include "shared/secondary/spp_worker_th/ringlatencystats.h"
+#include "shared/secondary/spp_worker_th/latency_stats.h"
 #endif
 
 /* Declare global variables */
diff --git a/src/shared/secondary/spp_worker_th/ringlatencystats.c b/src/shared/secondary/spp_worker_th/latency_stats.c
similarity index 99%
rename from src/shared/secondary/spp_worker_th/ringlatencystats.c
rename to src/shared/secondary/spp_worker_th/latency_stats.c
index b0f2080..7ca7385 100644
--- a/src/shared/secondary/spp_worker_th/ringlatencystats.c
+++ b/src/shared/secondary/spp_worker_th/latency_stats.c
@@ -14,7 +14,7 @@
 #include <rte_malloc.h>
 #include <rte_memcpy.h>
 
-#include "ringlatencystats.h"
+#include "latency_stats.h"
 #include "cmd_utils.h"
 #include "port_capability.h"
 #include "../return_codes.h"
diff --git a/src/shared/secondary/spp_worker_th/ringlatencystats.h b/src/shared/secondary/spp_worker_th/latency_stats.h
similarity index 100%
rename from src/shared/secondary/spp_worker_th/ringlatencystats.h
rename to src/shared/secondary/spp_worker_th/latency_stats.h
diff --git a/src/shared/secondary/spp_worker_th/port_capability.c b/src/shared/secondary/spp_worker_th/port_capability.c
index d4276f1..e90fd0b 100644
--- a/src/shared/secondary/spp_worker_th/port_capability.c
+++ b/src/shared/secondary/spp_worker_th/port_capability.c
@@ -12,7 +12,7 @@
 #include "shared/secondary/return_codes.h"
 
 #ifdef SPP_RINGLATENCYSTATS_ENABLE
-#include "ringlatencystats.h"
+#include "latency_stats.h"
 #endif
 
 /**
diff --git a/src/vf/Makefile b/src/vf/Makefile
index 784d636..742986c 100644
--- a/src/vf/Makefile
+++ b/src/vf/Makefile
@@ -20,7 +20,7 @@ SRCS-y += $(SPP_SEC_DIR)/string_buffer.c
 SRCS-y += $(SPP_SEC_DIR)/json_helper.c
 SRCS-y += $(SPP_SEC_DIR)/common.c
 SRCS-y += $(SPP_SEC_DIR)/utils.c $(SPP_SEC_DIR)/add_port.c
-SRCS-y += $(SPP_WKT_DIR)/ringlatencystats.c
+SRCS-y += $(SPP_WKT_DIR)/latency_stats.c
 SRCS-y += $(SPP_WKT_DIR)/port_capability.c
 SRCS-y += $(SPP_WKT_DIR)/conn_spp_ctl.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_parser.c
diff --git a/src/vf/classifier.c b/src/vf/classifier.c
index 375f710..fa47ed7 100644
--- a/src/vf/classifier.c
+++ b/src/vf/classifier.c
@@ -30,7 +30,7 @@
 #include "shared/secondary/spp_worker_th/port_capability.h"
 
 #ifdef SPP_RINGLATENCYSTATS_ENABLE
-#include "shared/secondary/spp_worker_th/ringlatencystats.h"
+#include "shared/secondary/spp_worker_th/latency_stats.h"
 #endif
 
 #define RTE_LOGTYPE_VF_CLS RTE_LOGTYPE_USER1
diff --git a/src/vf/forwarder.c b/src/vf/forwarder.c
index 3e98526..04bf5c3 100644
--- a/src/vf/forwarder.c
+++ b/src/vf/forwarder.c
@@ -10,7 +10,7 @@
 #include "shared/secondary/spp_worker_th/port_capability.h"
 
 #ifdef SPP_RINGLATENCYSTATS_ENABLE
-#include "shared/secondary/spp_worker_th/ringlatencystats.h"
+#include "shared/secondary/spp_worker_th/latency_stats.h"
 #endif
 
 #define RTE_LOGTYPE_FORWARD RTE_LOGTYPE_USER1
diff --git a/src/vf/spp_vf.c b/src/vf/spp_vf.c
index d6e6309..f843332 100644
--- a/src/vf/spp_vf.c
+++ b/src/vf/spp_vf.c
@@ -20,7 +20,7 @@
 #define RTE_LOGTYPE_SPP_VF RTE_LOGTYPE_USER1
 
 #ifdef SPP_RINGLATENCYSTATS_ENABLE
-#include "shared/secondary/spp_worker_th/ringlatencystats.h"
+#include "shared/secondary/spp_worker_th/latency_stats.h"
 #endif
 
 /* getopt_long return value for long option */
-- 
2.17.1

